src/Geolocation/Geolocation.md

```diff
@@ -10,9 +10,23 @@ Every time a library has a new release we should make an update.
 All releases are listed here: https://github.com/maxmind/GeoIP2-php/releases
 
 Files used:
-* geoip.mmdb
 * geoio.phar
 
+Updated:
+* 2022-08-05 - 2.13.0
+
+## Maxmind GeoLite2 Country db
```

Comment:
Might not be a bad idea to have an action that will check the '_up to dateness_' of the geiop.phar file.

Requests library does this to fetch the certificate bundle when the new ones are available: https://github.com/WordPress/Requests/blob/develop/.github/workflows/update-cacert.yml

What this workflow does is: it checks if the new bundle is available and will automatically create a PR for it. 

#AutomateAllTheThings 😄

Reply:
pr open